site/includes/exampleContent.pug

```diff
@@ -1,6 +1,9 @@
+-
+  const withThemeCompatibleLinks = (markdown) => markdown.replace(/<a href/g, '<a class="spectrum-Link" href');
```

Comment:
Doesn't this end up being a general problem for any markdown links rendered? I think instead of trying to inject the change here, maybe we just update the links in the docs to use the right class? We'll be fixing this programmatically with some of the markdown updates 11ty will bring with it.

Reply:
Yeah it's a problem for any that don't manually include the class. My thought was that doing it programmatically like this avoids human error, but if it'll be fixed in 11ty anyway then yeah I can adjust the links in the docs instead

Reply:
Ohhh it looks like Pug is handling this already when we use markdown instead of HTML (Accordion usage notes as an example). Should I spend the time now to update all of those mod links to use markdown?